packages/flutter/test/widgets/widget_inspector_test.dart

```diff
@@ -4160,8 +4168,8 @@ class _TestWidgetInspectorService extends TestWidgetInspectorService {
         equals('false'),
       );
       expect(extensionChangedEvents.length, equals(3));
-      expect(service.rebuildCount, equals(2));
-      expect(WidgetsApp.debugShowWidgetInspectorOverride, isFalse);
+      expect(debugShowChangeCounter, equals(2));
```

Comment:
wow this test had to be written in a wacky way before you change. thanks for cleaning this up!